calculator/viewCalculator.js

```diff
@@ -120,7 +120,7 @@ module.exports = calculateDuration;
 //   ]
 // };
 
-// calculateDuration(testObj);
+// calculateDuration(testObj.events);
```

Comment:
I really appreciate that you added a description about what this file does at the top of your file! Made it really easy to understand the purpose of your file. 

Maybe consider moving your obj that is commented out up to the top ? replace your first example obj
Loving all of the ES6 function instantiation, thinking we should make an agreement between our team that we use it whenever possible.

I would consider adding in a couple lines of sudo code within your function to  make it easier to follow

I would consider breaking out your orderEvents function into a helper function outside of your calculateDuration function- thinking it would make it easier to read!

Ok not sure if this is obvious but why are you dividing duration by 1000